phpBB/includes/mcp/mcp_main.php

```diff
@@ -174,7 +174,10 @@ function main($id, $mode)
 		switch ($mode)
 		{
 			case 'front':
-				include($phpbb_root_path . 'includes/mcp/mcp_front.' . $phpEx);
+				if (!function_exists('mcp_front_view'))
+				{
+					mcp_front_view($id, $mode, $action);
```

Comment:
Did you mean for this line to be `include($phpbb_root_path . 'includes/mcp/mcp_front.' . $phpEx);` ❓

Reply:
Indeed, thank you!